backend/src/Manager/Admin/StoreOwnerPayment/AdminStoreOwnerPaymentManager.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Manager\Admin\StoreOwnerPayment;
+
+use App\AutoMapping;
+use App\Entity\StoreOwnerPaymentEntity;
+use App\Repository\StoreOwnerPaymentEntityRepository;
+use App\Request\Admin\StoreOwnerPayment\AdminStoreOwnerPaymentCreateRequest;
+use App\Request\Admin\StoreOwnerPayment\AdminStoreOwnerPaymentUpdateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+use App\Manager\StoreOwner\StoreOwnerProfileManager;
+
+class AdminStoreOwnerPaymentManager
+{
+    private AutoMapping $autoMapping;
+    private EntityManagerInterface $entityManager;
+    private StoreOwnerPaymentEntityRepository $storeOwnerPaymentEntityRepository;
+    private StoreOwnerProfileManager $storeOwnerProfileManager;
+
+    public function __construct(AutoMapping $autoMapping, EntityManagerInterface $entityManager, StoreOwnerPaymentEntityRepository $storeOwnerPaymentEntityRepository, StoreOwnerProfileManager $storeOwnerProfileManager)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->entityManager = $entityManager;
+        $this->storeOwnerPaymentEntityRepository = $storeOwnerPaymentEntityRepository;
+        $this->storeOwnerProfileManager = $storeOwnerProfileManager;
+    }
+
+    public function createStoreOwnerPayment(AdminStoreOwnerPaymentCreateRequest $request): StoreOwnerPaymentEntity
+    {
+        $request->setStore($this->storeOwnerProfileManager->getStoreOwnerProfile($request->getStore()));
+
+        $storeOwnerPaymentEntity = $this->autoMapping->map(AdminStoreOwnerPaymentCreateRequest::class, StoreOwnerPaymentEntity::class, $request);
+
+        $this->entityManager->persist($storeOwnerPaymentEntity);
+        $this->entityManager->flush();
+
+        return $storeOwnerPaymentEntity;
+    }
+
+    public function updateStoreOwnerPayment(AdminStoreOwnerPaymentUpdateRequest $request): ?StoreOwnerPaymentEntity 
+    {
+        $storeOwnerPaymentEntity = $this->storeOwnerPaymentEntityRepository->find($request->getId());
+
+        if ($storeOwnerPaymentEntity) {
+   
+            $request->setStore($this->storeOwnerProfileManager->getStoreOwnerProfile($request->getStore()));
```

Comment:
Here you can set the store from the storeOwnerPaymentEntity while it is exist, right?